bigbluebutton-html5/imports/ui/components/chat/message-list/message-list-item/styles.scss

```diff
@@ -16,24 +16,11 @@
 }
 
 .systemMessage {
-  padding: var(--line-height-computed) 0;
-  border-bottom: 1px solid var(--color-gray-lighter);
-
-  .item:not(&) + & {
-    border-top: 1px solid var(--color-gray-lighter);
-  }
-
-  & + & {
-    margin-top: calc(var(--line-height-computed) * -1);
-  }
-
-  &:last-child {
-    border-bottom: none;
-  }
-
-  .message {
-    color: var(--color-gray);
-  }
+  background: #F9FBFC;
```

Comment:
The colors could be constants. The font-weight is also defined as `--btn-font-weight:`